ursula_cli/shell.py

```diff
@@ -27,8 +28,10 @@
 
 import yaml
 
+from distutils.version import LooseVersion
+
 LOG = logging.getLogger(__name__)
-ANSIBLE_VERSION = '1.9.2-bbg'
+ANSIBLE_VERSION = '1.9'
```

Comment:
why do we even need this anymore? I would prefer to push the validation logic to something in the ansible repo itself ...  a manifest type thing that's been talked about before.   In the meantime something like this could suffice - https://github.com/blueboxgroup/ursula-cli/pull/36